threatmodel.md

```diff
@@ -0,0 +1,6 @@
+Think of another threat model, and write a summary of it as if you were going to send it as a professional email to explain it to a coworker. Suggested ~1-2 paragraphs length, and addressing the core concepts (adversaries, attack surface and vector, and what sort of specific attacks and risks the user faces, suggested mitigations, etc.).
+
+Secret Gists
+
+Hey Coworker guy,
+Our team has been working on a program that uses API endpoints to programatically create secret (private + encoded) gists on GitHub.
```

Comment:
I'm guessing you worked in a group and/or had more that didn't make it here. I'll go ahead and close out the PR but if you want to push/reopen for more feedback that's fine - thanks!